### PR TITLE
QOL tweaks for system test setup

### PR DESCRIPTION
**Pull request by dhyon**
- main change is that we should be using the ds-test file in our lib folder rather than our own copy-pasta'd version

### CODE REVIEW

**Comment by drgorillamd** [comment]
Shouldn't we seize the opportunity to label addresses (there is a new cheatcode for this;)?

**Comment by dhyon** [comment]
> Shouldn't we seize the opportunity to label addresses (there is a new cheatcode for this;)?

sure I'm open to any improvements. could you explain more?

File: README.md [hunk 1]
```diff
@@ -31,11 +31,18 @@ forge build
...
 
 ```bash
-forge test
+forge test -vvv
+```
+
+4. Update Forge periodically:
+
+```bash
+forge update
```
Comment (drgorillamd) [inline README.md:40]: foundryup rather ?

File: README.md [hunk 2]
```diff
@@ -31,11 +31,18 @@ forge build
 3. Run tests:
 
 ```bash
-forge test
+forge test -vvv
```
Comment (drgorillamd) [inline README.md:34]: add `forge install ds-test` if we want to use the test.sol from lib/

**Comment by drgorillamd** [comment]
> > Shouldn't we seize the opportunity to label addresses (there is a new cheatcode for this;)?
> 
> sure I'm open to any improvements. could you explain more?

using `function label(address, string calldata) external;`, we could label all the contract/multisig by a custom label (new cheatcode in hevm)

File: README.md (hunk 2 above)
Comment (dhyon) [reply README.md:34]: gotcha I'll add `git submodule update --init` which is a bit more generic in case we add more submodules later down the line

**Pull request merged**